### ISSUE TITLE
Include imports in examples

### ISSUE THREAD

**Issue by MartinSStewart** [andrewMacmurray/elm-simple-animation#6]
A few of the code examples have things like `P.x` in them. I had no idea what this was until I opened up the documentation on elm-packages and saw that one of the modules is `Simple.Animation.Property`. From there I inferred that P was an alias for `Simple.Animation.Property`. 

I think it would be easier if, for example,
```elm
animatedDot : Html msg
animatedDot =
    Animated.div expandFade [ class "dot" ] []


expandFade : Animation
expandFade =
    Animation.fromTo
        { duration = 2000
        , options = [ Animation.loop ]
        }
        [ P.opacity 1, P.scale 1 ]
        [ P.opacity 0, P.scale 2 ]
```
was instead 
```elm
import Simple.Animated as Animated
import Simple.Animation as Animation
import SimpleAnimation.Property as P

animatedDot : Html msg
animatedDot =
    Animated.div expandFade [ class "dot" ] []


expandFade : Animation
expandFade =
    Animation.fromTo
        { duration = 2000
        , options = [ Animation.loop ]
        }
        [ P.opacity 1, P.scale 1 ]
        [ P.opacity 0, P.scale 2 ]
```

**Comment by andrewMacmurray** [comment]
Good idea, happy for you to make a PR if you have time?

**Comment by andrewMacmurray** [comment]
closed with #7
